norms.md

```diff
@@ -0,0 +1,22 @@
+##Fishtown Analytics Norms##
+
+###Listen, then speak.###
```

Comment:
Maybe a better way to describe what I'm trying to communicate is "Listen well." Or "Commit to understanding."